src/ripple/app/tests/PayChan_test.cpp

```diff
@@ -668,6 +668,68 @@ struct PayChan_test : public beast::unit_test::suite
         expect (chan1 != chan2);
     }
 
+    void
+    testRPC ()
+    {
+        testcase ("RPC");
+        using namespace jtx;
+        using namespace std::literals::chrono_literals;
+        Env env (*this, features (featurePayChan));
+        auto const alice = Account ("alice");
+        auto const bob = Account ("bob");
+        env.fund (XRP (10000), alice, bob);
+        auto const pk = alice.pk ();
+        auto const settleDelay = 3600s;
+        auto const channelFunds = XRP (1000);
+        env (create (alice, bob, channelFunds, settleDelay, pk));
+        env.close();
+        auto const chan1Str = to_string (channel (*env.current (), alice, bob));
+        std::string chan1PkStr;
+        {
+            auto const r =
+                env.rpc ("account_channels", alice.human (), bob.human ());
+            expect (r[jss::result][jss::channels].size () == 1);
+            expect (r[jss::result][jss::channels][0u][jss::channel_id] == chan1Str);
+            chan1PkStr = r[jss::result][jss::channels][0u][jss::public_key].asString();
+        }
+        {
+            auto const r =
+                env.rpc ("account_channels", bob.human (), alice.human ());
+            expect (r[jss::result][jss::channels].size () == 0);
+        }
+        env (create (alice, bob, channelFunds, settleDelay, pk));
+        env.close();
+        auto const chan2Str = to_string (channel (*env.current (), alice, bob));
+        {
+            auto const r =
+                    env.rpc ("account_channels", alice.human (), bob.human ());
+            expect (r[jss::result][jss::channels].size () == 2);
+            expect (chan1Str != chan2Str);
+            for(std::uint32_t i = 0; i<2; ++i)
```

Comment:
I'd prefer the following. Especially since chas2Str can be absent and the loop would pass.
expect(r[jss::result][jss::channels][0u][jss::channel_id] == chan2Str ||
            r[jss::result][jss::channels][1u][jss::channel_id] == chan2Str);